DESCRIPTION

```diff
@@ -29,71 +29,64 @@ Depends:
 Imports:
     mia
 Suggests:
-    rebook, 
-    glue, 
-    sessioninfo, 
-    miaTime, 
-    microbiomeDataSets,
-    microbiome,
-    curatedMetagenomicData, 
-    ggplot2, 
-    BiocParallel, 
-    Biostrings, 
-    phyloseq, 
-    gtools, 
-    purrr, 
-    vegan, 
-    fido,
-    rgl, 
-    plotly, 
-    knitr, 
-    dplyr, 
-    scales, 
-    DT, 
-    scater, 
-    bluster, 
-    patchwork, 
-    miaViz, 
-    ggsignif, 
-    pheatmap, 
-    ape, 
-    ggtree, 
-    kableExtra, 
-    dendextend, 
-    NbClust, 
-    cobiclust, 
-    biclust, 
-    ALDEx2, 
-    tidyverse, 
-    ANCOMBC, 
-    Maaslin2, 
-    MicrobiomeStat, 
-    GUniFrac, 
-    randomcoloR,
-    DirichletMultinomial, 
-    mikropml,
-    MLeval, 
-    sechm, 
-    ggpubr, 
-    stringr, 
-    ComplexHeatmap, 
-    reticulate,
-    igraph, 
-    corpcor, 
-    circlize, 
-    BiocBook,
-    BiocManager,
-    stats,
-    reshape2,
-    devtools,
-    NetCoMi,
-    SpiecEasi,
-    SPRING,
-    RColorBrewer,
-    factoextra,
-    grid,
-    cluster,
-    dada2
+	ALDEx2, 
+	ANCOMBC, 
+	ape, 
+	biclust, 
+	BiocBook,
+	BiocManager,
+	BiocParallel,
+	Biostrings, 	
+	bluster, 
+	circlize, 
+	cluster,
+	cobiclust, 
+	ComplexHeatmap, 
+	corpcor, 
+	curatedMetagenomicData, 
+	dada2,
+	dendextend, 
+	devtools,
+	dirichletMultinomial, 
+	DT,
+	factoextra,	
+	fido,
+	GGplot2, 
+	ggpubr, 
+	ggsignif, 
+	ggtree, 
+	glue, 
+	grid, 
+	igraph, 
+	kableExtra, 
+	knitr, 
+	miaTime, 
+	miaViz, 
+	Microbiome, 
+	microbiomeDataSets,
+	MicrobiomeStat,
+	Mikropml,	
+	MLeval, 
+	NbClust,
+	NetCoMi,
+	patchwork,
+	pheatmap, 
+	phyloseq, 
+	plotly, 
+	purrr, 
+	randomcoloR,
+	reshape2,
+	reticulate,
+	rgl, 
+	scales,
+	scater, 
+	sechm, 
+	sessioninfo, 
+	SPRING,
+	stats,
+	Stringr, 
+	tidyverse,
+	Vegan
```

Comment:
Check the correct writing of all the packages. At least: Vegan -> vegan, Stringr -> stringr

Also the list should be updated because there are now new packages in microbiome:master